EXT_LaserEnvelope.md

```diff
@@ -0,0 +1,68 @@
+Domain-Specific Naming Conventions for a laser envelope profile
+===============================================================
+
+openPMD extension name: `LaserEnvelope`
+
+
+Introduction
+------------
+
+This extension is specifically designed for the modeling laser beam propagation.
+
+Mesh Based Records (Fields)
+---------------------------
+
+### Naming Conventions for `mesh record`s (field records)
+
+The laser pulse is represented as a single complex field $\mathcal{E}$, describing the envelope of the laser electric field in the paraxial approximation:
```

Comment:
```suggestion
The laser pulse is represented as a single complex field $\mathcal{E}$, describing the envelope of the laser electric field:
```
My understanding is that we do not need to do the paraxial approximation at this point. In fact, the propagator we will implement in `lasy` may not make this approximation.

Reply:
I think that's correct